src/components/ConfirmModalForm.js

```diff
@@ -5,41 +5,22 @@ import { Form } from "reactstrap";
 import i18n from "../login/translation/InjectIntl_HOC_factory";
 import CustomInput from "../login/components/Inputs/CustomInput";
 
+// Check the user supplied value against a regexp pattern provided when creating the ConfirmModal.
 const validate = (values, props) => {
```

Comment:
Are we still planning to move the validation functions out of the components into separate files? 
I have some reasons for why I think we should do it and (if there are any) I'd be happy to hear any reasons why they should remain in the file.

Reply:
Are you proposing to duplicate the code into each file that has a modal in it? I would say that is bad practice. You almost always end up with different bugs fixed in the different incarnations of the code.

https://en.wikipedia.org/wiki/Don%27t_repeat_yourself